app/javascript/onboarding/utilities.js

```diff
@@ -6,3 +6,16 @@ export const jsonToForm = data => {
 
 export const getContentOfToken = token =>
   document.querySelector(`meta[name='${token}']`).content;
+
+export const stringfy = text => {
```

Comment:
I think this function should have a more meaningful name, maybe `updateOnboarding`, and the parameter name could be called `lastPage`
🤔

Reply:
This is actually a good advice. I made the function with the ideia to change it's name later and totally forgot :v
My bad :sweat_smile:

Reply:
Also, I will keep an eye for things like this in the future. Thanks for the help :smile: